pandas/tests/io/formats/data/gh21625_expected_output.html

```diff
@@ -0,0 +1,14 @@
+<table border="1" class="dataframe">
```

Comment:
Don't have a very strong opinion atm but wondering if we want dedicated files for tests this small. Could maybe include HTML as part of the test or alternately just test for the existence of the appropriate format in the result or the expression.

This doesn't need to be addressed in this PR but just bringing up as a general discussion point